diff --git a/node_modules/expo-file-system/ios/FileSystemModule.swift b/node_modules/expo-file-system/ios/FileSystemModule.swift
index 3519c67..cc6a9cc 100644
--- a/node_modules/expo-file-system/ios/FileSystemModule.swift
+++ b/node_modules/expo-file-system/ios/FileSystemModule.swift
@@ -7,7 +7,7 @@ private let EVENT_DOWNLOAD_PROGRESS = "expo-file-system.downloadProgress"
 private let EVENT_UPLOAD_PROGRESS = "expo-file-system.uploadProgress"
 
 public final class FileSystemModule: Module {
-  private lazy var sessionTaskDispatcher = EXSessionTaskDispatcher(sessionHandler: ExpoAppDelegate.getSubscriberOfType(FileSystemBackgroundSessionHandler.self))
+  private lazy var sessionTaskDispatcher = EXSessionTaskDispatcher(sessionHandler: nil)
   private lazy var taskHandlersManager = EXTaskHandlersManager()
   private lazy var resourceManager = PHAssetResourceManager()
 
